packages/input/src/Input.js

```diff
@@ -85,34 +77,10 @@ Input.propTypes = {
    * Prevents the user from interacting with the input
    */
   disabled: PropTypes.bool,
-  /**
-   * HTML ID attribute
-   */
-  id: PropTypes.string,
-  /**
-   * A hint to browsers for which virtual keyboard to display
-   */
-  inputMode: PropTypes.oneOf(availableInputModes),
-  /**
-   * The maximum number of characters that the user can enter
-   */
-  maxLength: PropTypes.string,
-  /**
-   * The minimum number of characters that the user can enter
-   */
-  minLength: PropTypes.string,
-  /**
-   * Submitted with the control's value as part of the form data
-   */
-  name: PropTypes.string,
   /**
    * Fired when an element has lost focus
    */
   onBlur: PropTypes.func,
-  /**
-   * Fired when the value is changed by the user
-   */
-  onChange: PropTypes.func,
```

Comment:
do we want to keep `onChange` here? it was the only event handler removed, but is also still part in the storybook example and the action logger is firing for it.

Reply:
I meant to leave it off. 

This does make for weird docs that only show a few seemingly arbitrary props, but we pass everything thought, and we can't document everything. I'll add a note to the Readme to look at MDN for the full set of attributes an input takes.